caffe2/operators/group_norm_op.cc

```diff
@@ -1,16 +1,196 @@
-// Copyright 2004-present Facebook. All Rights Reserved.
-
 #include "group_norm_op.h"
 
+#include <array>
+
+#include "caffe2/utils/math.h"
+
 namespace caffe2 {
 
+namespace {
+
+template <typename T>
+inline T Cube(const T& x) {
```

Comment:
Oh, it's inline, probably no need.

Reply:
Yes, and this one is a very simple function and Cube is not very popular in our use case. So I think there is no need to move it.